crates/provider-sdk/wit/deps.toml

```diff
@@ -0,0 +1,2 @@
+logging = "https://github.com/WebAssembly/wasi-logging/archive/main.tar.gz"
+wasmcloud-core = "."
```

Comment:
Why would this ever be necessary?

Reply:
I was just copy pasting from previous examples